{"name": "test_add_2", "status": "failed", "statusDetails": {"message": "AssertionError: 0.003 != 0.03", "trace": "self = <testing.test_calc.TestCalc testMethod=test_add_2>\n\n    def test_add_2(self):\n>       self.assertEqual(0.003, self.calc.add(0.01, 0.02))\nE       AssertionError: 0.003 != 0.03\n\ntest_calc.py:16: AssertionError"}, "start": 1586331224577, "stop": 1586331224577, "uuid": "71e53d53-d597-4124-b7d5-6caf79f3abf4", "historyId": "0acd38cb0f4196350e91118baa263082", "testCaseId": "83d1502ce6bb66abac2a7c92fd0b8cb2", "fullName": "test_calc.TestCalc#test_add_2", "labels": [{"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-8G9LOH5"}, {"name": "thread", "value": "9700-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}